src/Services/Twilio/TwilioHttpClient.php

```diff
@@ -23,9 +24,10 @@ public function request(
         array $params = [],
         array $data = [],
         array $headers = [],
-        string|null $user = null,
-        string|null $password = null,
-        int|null $timeout = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?int $timeout = null,
+        ?AuthStrategy $authStrategy = null
```

Comment:
Just to maintain our style guidelines:
```suggestion
        string|null $user = null,
        string|null $password = null,
        int|null $timeout = null,
        AuthStrategy|null $authStrategy = null
```